src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -147,24 +147,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
                    action='store_true',
                    arg_group='Open Restriction',
                    help='Use open restriction. License or key will be delivered on every request. Not recommended for production environments.')
-        c.argument('alt_key_symmetric',
-                   action='store_true',
-                   arg_group='Token Restriction Key Type',
-                   help='Use a symmetric key as an alternate token key.')
-        c.argument('alt_key_rsa',
-                   action='store_true',
-                   arg_group='Token Restriction Key Type',
-                   help='Use an RSA public key as an alternate token key.')
-        c.argument('alt_key_x509',
-                   action='store_true',
-                   arg_group='Token Restriction Key Type',
-                   help='Use a certificate in PKCS12 format (X509Certificate2 in .NET) as an alternate token key.')
         c.argument('policy_option_name', help='The content key policy option name.')
         c.argument('policy_option_id', help='The content key policy option identifier.')
         c.argument('issuer', arg_group='Token Restriction', help='The token issuer.')
         c.argument('audience', arg_group='Token Restriction', help='The audience for the token.')
-        c.argument('token_key', arg_group='Token Restriction', help='Either a string (for symmetric key) or a filepath to a certificate (x509) or public key (rsa). Must be used with a Token Restriction Key Type flag.')
+        c.argument('token_key', arg_group='Token Restriction', help='Either a string (for symmetric key) or a filepath to a certificate (x509) or public key (rsa). To be used with token_key_type.')
```

Comment:
`Must be used in conjunction with --token-key-type` wouldn't be better?. Same goes for alternate argument.

Reply:
Fixed